vignettes/folder.Rmd

```diff
@@ -0,0 +1,88 @@
+---
+title: "Recommended folder structure"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Recommended folder structure}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## For a simple R project
+
+`checklist` assumes you have some main folder for your project.
+Within that main folder you create the `checklist` project which should contain all R related files (scripts and data).
+Place all non R related files in other folders under the main folder.
+
+The `checklist` project has a mandatory folder `source` where you place all the R and Rmarkdown files.
+The other folders are optional.
+The `data` folder holds all the data files for the analysis.
+The `output` folder holds the rendered files from the analysis.
+The `media` folder holds the images you want to incorporate in the Rmarkdown files.
+The `inst` folder stores some `checklist` specific data.
+
+While you can add other folders the root of your `checklist` project, we recommend to stick with the default folders.
+Having a non standard folder name results in a warning when running `check_folder()`.
```

Comment:
```suggestion
Having a nonstandard folder name results in a warning when running `check_folder()`.
```

Reply:
fixed in [60f3b39](https://github.com/inbo/checklist/pull/94/commits/60f3b395f45b74f7ff493bf9d73e8a7530419d3a)